<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="7" total="30" passed="23">
  <reporter-output>
  </reporter-output>
  <suite name="All Test Suite" duration-ms="803907" started-at="2019-03-11T08:58:43Z" finished-at="2019-03-11T09:12:07Z">
    <groups>
    </groups>
    <test name="C:/Users/jtc/IdeaProjects/game" duration-ms="803907" started-at="2019-03-11T08:58:43Z" finished-at="2019-03-11T09:12:07Z">
      <class name="AgentManagement.creditDetail">
        <test-method status="PASS" signature="creditExplorerNumber()[pri:23, instance:AgentManagement.creditDetail@727453a3]" name="creditExplorerNumber" duration-ms="24217" started-at="2019-03-11T16:08:21Z" finished-at="2019-03-11T16:08:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- creditExplorerNumber -->
        <test-method status="FAIL" signature="creditExplorerServiceHistoryTotal()[pri:24, instance:AgentManagement.creditDetail@727453a3]" name="creditExplorerServiceHistoryTotal" duration-ms="27374" started-at="2019-03-11T16:08:45Z" finished-at="2019-03-11T16:09:12Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at AgentManagement.creditDetail.creditExplorerServiceHistoryTotal(creditDetail.java:97)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- creditExplorerServiceHistoryTotal -->
        <test-method status="PASS" signature="creditExplorerAccountBalance()[pri:25, instance:AgentManagement.creditDetail@727453a3]" name="creditExplorerAccountBalance" duration-ms="27576" started-at="2019-03-11T16:09:12Z" finished-at="2019-03-11T16:09:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- creditExplorerAccountBalance -->
        <test-method status="PASS" signature="creditBusinessNumber()[pri:26, instance:AgentManagement.creditDetail@727453a3]" name="creditBusinessNumber" duration-ms="26254" started-at="2019-03-11T16:09:40Z" finished-at="2019-03-11T16:10:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- creditBusinessNumber -->
        <test-method status="FAIL" signature="creditBusinessServiceHistoryTotal()[pri:27, instance:AgentManagement.creditDetail@727453a3]" name="creditBusinessServiceHistoryTotal" duration-ms="33515" started-at="2019-03-11T16:10:06Z" finished-at="2019-03-11T16:10:40Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at AgentManagement.creditDetail.creditBusinessServiceHistoryTotal(creditDetail.java:239)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- creditBusinessServiceHistoryTotal -->
        <test-method status="FAIL" signature="creditBusinessServiceAccountBalance()[pri:28, instance:AgentManagement.creditDetail@727453a3]" name="creditBusinessServiceAccountBalance" duration-ms="30272" started-at="2019-03-11T16:10:40Z" finished-at="2019-03-11T16:11:10Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at AgentManagement.creditDetail.creditBusinessServiceAccountBalance(creditDetail.java:289)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- creditBusinessServiceAccountBalance -->
        <test-method status="FAIL" signature="creditAgentServiceHistoryTotal()[pri:29, instance:AgentManagement.creditDetail@727453a3]" name="creditAgentServiceHistoryTotal" duration-ms="28567" started-at="2019-03-11T16:11:10Z" finished-at="2019-03-11T16:11:38Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at AgentManagement.creditDetail.creditAgentServiceHistoryTotal(creditDetail.java:338)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- creditAgentServiceHistoryTotal -->
        <test-method status="FAIL" signature="creditAgentServiceAccountBalance()[pri:30, instance:AgentManagement.creditDetail@727453a3]" name="creditAgentServiceAccountBalance" duration-ms="28638" started-at="2019-03-11T16:11:38Z" finished-at="2019-03-11T16:12:07Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at AgentManagement.creditDetail.creditAgentServiceAccountBalance(creditDetail.java:387)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- creditAgentServiceAccountBalance -->
      </class> <!-- AgentManagement.creditDetail -->
      <class name="AgentManagement.CashAgentDetail">
        <test-method status="PASS" signature="immediateLowerAgentNumber()[pri:15, instance:AgentManagement.CashAgentDetail@3386ed8]" name="immediateLowerAgentNumber" duration-ms="26982" started-at="2019-03-11T16:04:03Z" finished-at="2019-03-11T16:04:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- immediateLowerAgentNumber -->
        <test-method status="PASS" signature="DirectPushPlayerNumber()[pri:16, instance:AgentManagement.CashAgentDetail@3386ed8]" name="DirectPushPlayerNumber" duration-ms="24985" started-at="2019-03-11T16:04:30Z" finished-at="2019-03-11T16:04:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DirectPushPlayerNumber -->
        <test-method status="PASS" signature="taxTotal()[pri:17, instance:AgentManagement.CashAgentDetail@3386ed8]" name="taxTotal" duration-ms="33348" started-at="2019-03-11T16:04:55Z" finished-at="2019-03-11T16:05:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- taxTotal -->
        <test-method status="PASS" signature="promotionCommissionTotal()[pri:18, instance:AgentManagement.CashAgentDetail@3386ed8]" name="promotionCommissionTotal" duration-ms="34438" started-at="2019-03-11T16:05:29Z" finished-at="2019-03-11T16:06:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- promotionCommissionTotal -->
        <test-method status="PASS" signature="acceptRedEnvelopeTotal()[pri:19, instance:AgentManagement.CashAgentDetail@3386ed8]" name="acceptRedEnvelopeTotal" duration-ms="32541" started-at="2019-03-11T16:06:03Z" finished-at="2019-03-11T16:06:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- acceptRedEnvelopeTotal -->
        <test-method status="PASS" signature="addWithdrawalMoney()[pri:20, instance:AgentManagement.CashAgentDetail@3386ed8]" name="addWithdrawalMoney" duration-ms="35218" started-at="2019-03-11T16:06:36Z" finished-at="2019-03-11T16:07:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addWithdrawalMoney -->
        <test-method status="PASS" signature="commissionAccountBalance()[pri:21, instance:AgentManagement.CashAgentDetail@3386ed8]" name="commissionAccountBalance" duration-ms="35047" started-at="2019-03-11T16:07:11Z" finished-at="2019-03-11T16:07:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- commissionAccountBalance -->
        <test-method status="PASS" signature="rechargeBalance()[pri:22, instance:AgentManagement.CashAgentDetail@3386ed8]" name="rechargeBalance" duration-ms="34679" started-at="2019-03-11T16:07:46Z" finished-at="2019-03-11T16:08:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- rechargeBalance -->
      </class> <!-- AgentManagement.CashAgentDetail -->
      <class name="AgentManagement.CashBusinessDetail">
        <test-method status="FAIL" signature="immediateLowerAgentNumber()[pri:8, instance:AgentManagement.CashBusinessDetail@598778cc]" name="immediateLowerAgentNumber" duration-ms="30464" started-at="2019-03-11T16:00:43Z" finished-at="2019-03-11T16:01:13Z">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id="root"]/section/section/main/div[2]/div[2]/div/div[1]/div/div[2]/div/div[3]/table/tbody/tr[2]/td[3]/div/a"}
  (Session info: chrome=72.0.3626.121)
  (Driver info: chromedriver=2.45.615291 (ec3682e3c9061c10f26ea9e5cdcf3c53f3f74387),platform=Windows NT 10.0.17134 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 10.01 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.0.0', revision: '350cf60', time: '2016-10-13 10:48:57 -0700'
System info: host: 'DESKTOP-NDF1E9V', ip: '192.168.1.252', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_191'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{mobileEmulationEnabled=false, timeouts={implicit=0, pageLoad=300000, script=30000}, hasTouchScreen=false, platform=XP, acceptSslCerts=false, goog:chromeOptions={debuggerAddress=localhost:61142}, acceptInsecureCerts=false, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, setWindowRect=true, unexpectedAlertBehaviour=ignore, applicationCacheEnabled=false, rotatable=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.45.615291 (ec3682e3c9061c10f26ea9e5cdcf3c53f3f74387), userDataDir=C:\Users\jtc\AppData\Local\Temp\scoped_dir121508_21586}, takesHeapSnapshot=true, pageLoadStrategy=normal, strictFileInteractability=false, databaseEnabled=false, handlesAlerts=true, version=72.0.3626.121, browserConnectionEnabled=false, proxy={}, nativeEvents=true, locationContextEnabled=true, cssSelectorsEnabled=true}]
Session ID: 2ba9433523414dbe182069483af07b86
*** Element info: {Using=xpath, value=//*[@id="root"]/section/section/main/div[2]/div[2]/div/div[1]/div/div[2]/div/div[3]/table/tbody/tr[2]/td[3]/div/a}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id="root"]/section/section/main/div[2]/div[2]/div/div[1]/div/div[2]/div/div[3]/table/tbody/tr[2]/td[3]/div/a"}
  (Session info: chrome=72.0.3626.121)
  (Driver info: chromedriver=2.45.615291 (ec3682e3c9061c10f26ea9e5cdcf3c53f3f74387),platform=Windows NT 10.0.17134 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 10.01 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.0.0', revision: '350cf60', time: '2016-10-13 10:48:57 -0700'
System info: host: 'DESKTOP-NDF1E9V', ip: '192.168.1.252', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_191'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{mobileEmulationEnabled=false, timeouts={implicit=0, pageLoad=300000, script=30000}, hasTouchScreen=false, platform=XP, acceptSslCerts=false, goog:chromeOptions={debuggerAddress=localhost:61142}, acceptInsecureCerts=false, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, setWindowRect=true, unexpectedAlertBehaviour=ignore, applicationCacheEnabled=false, rotatable=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.45.615291 (ec3682e3c9061c10f26ea9e5cdcf3c53f3f74387), userDataDir=C:\Users\jtc\AppData\Local\Temp\scoped_dir121508_21586}, takesHeapSnapshot=true, pageLoadStrategy=normal, strictFileInteractability=false, databaseEnabled=false, handlesAlerts=true, version=72.0.3626.121, browserConnectionEnabled=false, proxy={}, nativeEvents=true, locationContextEnabled=true, cssSelectorsEnabled=true}]
Session ID: 2ba9433523414dbe182069483af07b86
*** Element info: {Using=xpath, value=//*[@id="root"]/section/section/main/div[2]/div[2]/div/div[1]/div/div[2]/div/div[3]/table/tbody/tr[2]/td[3]/div/a}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:216)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:168)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:635)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:368)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:473)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:360)
	at PublicMethod.WebDriverUtil.findElementByXpath(WebDriverUtil.java:192)
	at PublicMethod.WebDriverUtil.getTextByXpath(WebDriverUtil.java:449)
	at AgentManagement.CashBusinessDetail.immediateLowerAgentNumber(CashBusinessDetail.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- immediateLowerAgentNumber -->
        <test-method status="PASS" signature="taxTotal()[pri:9, instance:AgentManagement.CashBusinessDetail@598778cc]" name="taxTotal" duration-ms="29540" started-at="2019-03-11T16:01:13Z" finished-at="2019-03-11T16:01:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- taxTotal -->
        <test-method status="PASS" signature="alreadySettlementMoney()[pri:10, instance:AgentManagement.CashBusinessDetail@598778cc]" name="alreadySettlementMoney" duration-ms="27534" started-at="2019-03-11T16:01:43Z" finished-at="2019-03-11T16:02:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- alreadySettlementMoney -->
        <test-method status="PASS" signature="waitWithdrawMoney()[pri:11, instance:AgentManagement.CashBusinessDetail@598778cc]" name="waitWithdrawMoney" duration-ms="27794" started-at="2019-03-11T16:02:10Z" finished-at="2019-03-11T16:02:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- waitWithdrawMoney -->
        <test-method status="PASS" signature="rechargeBalance()[pri:12, instance:AgentManagement.CashBusinessDetail@598778cc]" name="rechargeBalance" duration-ms="30048" started-at="2019-03-11T16:02:38Z" finished-at="2019-03-11T16:03:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- rechargeBalance -->
        <test-method status="PASS" signature="onlineRechargeCredit()[pri:13, instance:AgentManagement.CashBusinessDetail@598778cc]" name="onlineRechargeCredit" duration-ms="28545" started-at="2019-03-11T16:03:08Z" finished-at="2019-03-11T16:03:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- onlineRechargeCredit -->
        <test-method status="PASS" signature="onlineWithdrawCredit()[pri:14, instance:AgentManagement.CashBusinessDetail@598778cc]" name="onlineWithdrawCredit" duration-ms="26507" started-at="2019-03-11T16:03:37Z" finished-at="2019-03-11T16:04:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- onlineWithdrawCredit -->
      </class> <!-- AgentManagement.CashBusinessDetail -->
      <class name="InbetAdminTestCase.AdminHomeTest">
        <test-method status="PASS" signature="serviceFeeIncome()[pri:1, instance:InbetAdminTestCase.AdminHomeTest@1e606f64]" name="serviceFeeIncome" duration-ms="19081" started-at="2019-03-11T15:58:43Z" finished-at="2019-03-11T15:59:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- serviceFeeIncome -->
        <test-method status="PASS" signature="sinceBusinessOperationsTotalArrears()[pri:2, instance:InbetAdminTestCase.AdminHomeTest@1e606f64]" name="sinceBusinessOperationsTotalArrears" duration-ms="17504" started-at="2019-03-11T15:59:02Z" finished-at="2019-03-11T15:59:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sinceBusinessOperationsTotalArrears -->
        <test-method status="PASS" signature="businessMoneyTotal()[pri:3, instance:InbetAdminTestCase.AdminHomeTest@1e606f64]" name="businessMoneyTotal" duration-ms="17432" started-at="2019-03-11T15:59:20Z" finished-at="2019-03-11T15:59:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- businessMoneyTotal -->
        <test-method status="PASS" signature="playerNumber()[pri:4, instance:InbetAdminTestCase.AdminHomeTest@1e606f64]" name="playerNumber" duration-ms="16607" started-at="2019-03-11T15:59:37Z" finished-at="2019-03-11T15:59:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- playerNumber -->
        <test-method status="FAIL" signature="systemWinOrLose()[pri:5, instance:InbetAdminTestCase.AdminHomeTest@1e606f64]" name="systemWinOrLose" duration-ms="15613" started-at="2019-03-11T15:59:54Z" finished-at="2019-03-11T16:00:10Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at InbetAdminTestCase.AdminHomeTest.systemWinOrLose(AdminHomeTest.java:161)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- systemWinOrLose -->
        <test-method status="PASS" signature="todayCommission()[pri:6, instance:InbetAdminTestCase.AdminHomeTest@1e606f64]" name="todayCommission" duration-ms="15527" started-at="2019-03-11T16:00:10Z" finished-at="2019-03-11T16:00:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- todayCommission -->
        <test-method status="PASS" signature="totalCommission()[pri:7, instance:InbetAdminTestCase.AdminHomeTest@1e606f64]" name="totalCommission" duration-ms="17698" started-at="2019-03-11T16:00:25Z" finished-at="2019-03-11T16:00:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- totalCommission -->
      </class> <!-- InbetAdminTestCase.AdminHomeTest -->
    </test> <!-- C:/Users/jtc/IdeaProjects/game -->
  </suite> <!-- All Test Suite -->
</testng-results>
